src/axom/sidre/core/Buffer.cpp

```diff
@@ -423,7 +423,7 @@ void* Buffer::allocateBytes(IndexType num_bytes, int allocID)
 {
   allocID = getValidAllocatorID(allocID);
 #ifdef AXOM_USE_UMPIRE
```

Comment:
This no longer needs to be guarded by ``AXOM_USE_UMPIRE`` -- it should just be 
```cpp
return axom::allocate<axom::int8>(num_bytes, allocID ); 
```

Reply:
Yup. Thanks for catching that. Done.